File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/Blood.cs
using HospitalLibrary.Core.Model.Enums;

namespace HospitalLibrary.Core.Model
{
    public class Blood: BaseModel
    {
        public BloodType BloodType { get; set; }
        public int Quantity { get; set; }

        public int RoomId { get; set; }

        public Blood() { }
        public Blood(int id, bool deleted, BloodType bloodType, int quantity)
        {
            Id = id;
            Deleted = deleted;
            BloodType = bloodType;
            Quantity = quantity;
        }

        public Blood(int id, BloodType bloodType, int quantity)
        {
            Deleted = false;
            BloodType = bloodType;
            Quantity = quantity;
            Id = id;
        }

    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/blood.ts
import { BloodType } from "./bloodType";

export class Blood {
    id: number;
    deleted: boolean;
    bloodType: BloodType;
    quantity: number = 0;

    public constructor(id: any, deleted: any, bloodType: any, quantity: any) {
        this.id = id;
        this.deleted = deleted;
        this.bloodType = bloodType;
        this.quantity = quantity;
    }
}